apps/webapp/app/presenters/v3/TestTaskPresenter.server.ts

```
@@ -143,58 +178,64 @@ export class TestTaskPresenter {
+                  metadata: r.seedMetadata
+                    ? await prettyPrintPacket(r.seedMetadata, r.seedMetadataType)
+                    : undefined,
+                };
               })
-            )
-          ).filter(Boolean),
+            ),
+          },
+        };
+      case "SCHEDULED":
+        const possibleTimezones = getTimezones();
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)